Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -743,7 +782,7 @@ def _create_incident_from_alert(self, alert):
         return incident
 
     def _relate_files_to_alert(self, alert_details):
-        for obs in alert_details["observations"]:
+        for obs in alert_details.get("observations"):
```

Comment:
If alert_details doesn't have an "observations" key this will still raise an exception because `None` isn't iterable.

Reply:
you can just make the default return value an empty list instead of `None` and it'll not actually process anything if "observations" doesn't exist. 

```
for obs in alert_details.get("observations", []):
```

Reply:
I don't think that will ever happen, but still good to handle better